features/product/associate_product.feature

```diff
@@ -29,38 +29,38 @@ Feature: Associate a product
   Scenario: Associate a product to another group
     Given I edit the "charcoal-boots" product
     When I visit the "Associations" column tab
-    And I select the "Upsell" association
+    And I visit the "Upsell" association type
     And I press the "Show groups" button
     And I check the row "caterpillar_boots"
     And I save the product
     And I edit the "charcoal-boots" product
     And I visit the "Associations" column tab
-    And I select the "Upsell" association
+    And I visit the "Upsell" association type
     Then I should see the text "0 products and 1 groups"
     And the row "caterpillar_boots" should be checked
 
   @jira https://akeneo.atlassian.net/browse/PIM-4788
   Scenario: Associate a product to multiple products and groups
     Given I edit the "black-boots" product
     And I visit the "Associations" column tab
-    And I select the "Substitution" association
+    And I visit the "Substitution" association type
     And I check the row "charcoal-boots"
-    And I select the "Upsell" association
+    And I visit the "Upsell" association type
     And I check the row "glossy-boots"
     And I press the "Show groups" button
     And I check the row "caterpillar_boots"
-    And I select the "Cross sell" association
+    And I visit the "Cross sell" association type
     And I check the row "similar_boots"
     And I press the "Show products" button
     And I check the rows "shoelaces, gray-boots, brown-boots and green-boots"
     When I save the product
     Then I should not see the text "There are unsaved changes."
     And I should see the text "4 products and 1 groups"
-    And I select the "Upsell" association
+    And I visit the "Upsell" association type
```

Comment:
I only changed the text, and this for several dozens of feature files.. This will be pain to fix, and don't change anything.
And please don't comment only by adding a comment ; do a complete review, give GTM or not GTM, but not single comment then jetpack please.